pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <artifactId>killbill-oss-parent</artifactId>
         <groupId>org.kill-bill.billing</groupId>
-        <version>0.67</version>
+        <version>0.68-SNAPSHOT</version>
```

Comment:
It contains the new version of killbill-client-java. PR: https://github.com/killbill/killbill-client-java/pull/22